src/SwedbankPay.Sdk.Infrastructure/Extensions/UriExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using SwedbankPay.Sdk.PaymentInstruments;
+using SwedbankPay.Sdk.PaymentOrders;
+using System;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.Extensions
+{
+    public static class UriExtensions
+    {
+        public static Uri GetUrlWithQueryString(this Uri uri, PaymentExpand paymentExpand)
+        {
+            var paymentExpandQueryString = GetExpandQueryString<PaymentExpand>(paymentExpand);
+            var url = !string.IsNullOrWhiteSpace(paymentExpandQueryString)
+                ? new Uri(uri.OriginalString + paymentExpandQueryString, UriKind.RelativeOrAbsolute)
+                : uri;
+            return url;
+        }
+
+        public static Uri GetUrlWithQueryString(this Uri uri, PaymentOrderExpand paymentExpand)
+        {
+            string paymentExpandQueryString = GetExpandQueryString<PaymentOrderExpand>(paymentExpand);
+            var url = !string.IsNullOrWhiteSpace(paymentExpandQueryString)
+                ? new Uri(uri.OriginalString + paymentExpandQueryString, UriKind.RelativeOrAbsolute)
+                : uri;
+            return url;
+        }
+
+        private static string GetExpandQueryString<T>(Enum paymentExpand)
+             where T : Enum
+        {
+            var intValue = Convert.ToInt64(paymentExpand);
+            if (intValue == 0)
+                return string.Empty;
+
+            var s = new List<string>();
+            foreach (var enumValue in Enum.GetValues(typeof(T)))
+            {
+                var name = Enum.GetName(typeof(T), enumValue);
+                if (paymentExpand.HasFlag((T)enumValue) && name != "None" && name != "All")

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)